### PR TITLE
Second round of improvements on documentation

### PR DESCRIPTION
**Pull request by williamjamir**
This pull request increments the documentation by:

- Fixing the logo image that was not showing at ReadTheDocs;
- Improving the `Quick Start` section, adding an example with C++
- Change application utility section

Link for the ReadTheDocs page of this version:
https://hookman.readthedocs.io/en/docs-improvements/

### CODE REVIEW

File: docs/source/concepts/application_usage.rst [hunk 1]
```diff
@@ -3,41 +3,18 @@ Application utilities
...
+- **Install Plugins**: 
+    Install a plugun by informing the path to where the ``hm.plugin`` is located.
+- **Remove a Plugin**: 
+    Remove the plugin by informing the name of the plugin.
+- **List all plugins availables**: 
+    Get a list of plugins with all the plugin details.
+- **Get the status of all plugins**: 
+    Check if there are conflicts between them.
+
+The method that list all plugins available return a list of ``PluginInfoo``, for more details about the :ref:`plugin-info-api-section`, read the API section.
```
Comment (tadeu) [inline docs/source/concepts/application_usage.rst]: ``PluginInfoo`` -> ``PluginInfo``

also, perhaps:

```
, for more details read the :ref:`plugin-info-api-section` API section.
```

File: docs/source/concepts/application_usage.rst [hunk 2]
```diff
@@ -3,41 +3,18 @@ Application utilities
...
-.. automethod:: hookman.hooks.HookMan.get_status
-
-
-GetHookCaller
--------------
-
-.. automethod:: hookman.hooks.HookMan.get_hook_caller
-
-
+With the ``HookMan`` you can:
```
Comment (tadeu) [inline docs/source/concepts/application_usage.rst]: Perhaps it's better to remote the `the`

File: docs/source/quick_start.rst [hunk 3]
```diff
@@ -89,7 +98,58 @@ The example below shows how to execute the method in a python module.
...
     #Executing the method implemented in one of the plugins.
-    friction_factor_function(argument1, argument2).
+    ff_result = friction_factor_function(1, 2.5).
+
+    print(f"Result from friction_factor hook: {ff_result}")
+
+Executing in C++
+--------------------
+
+As mentioned on the `pybind11 functional documentation`_, the C++11 standard brought the generic polymorphic function wrapper std::function<>
```
Comment (tadeu) [inline docs/source/quick_start.rst]: backticks around `std::function<>`?

File: docs/source/quick_start.rst [hunk 4]
```diff
@@ -89,7 +98,58 @@ The example below shows how to execute the method in a python module.
...
+   
+   #include <pybind11/functional.h>
+
+    PYBIND11_MODULE(my_cpp_binding_module, m) {
+        m.def("func_friction_factor", &friction_factor);
+
+    }
+
+The example below shows how to create an object ``hook_caller``, 
+and pass a function implemented on one of the plugins directly to C++ a fucntion.
```
Comment (tadeu) [inline docs/source/quick_start.rst]: fucntion -> function, also the comma before `and` may be removed

File: docs/source/concepts/application_usage.rst (hunk 1 above)
Comment (williamjamir) [reply docs/source/concepts/application_usage.rst]: Fixed ✔️

File: docs/source/concepts/application_usage.rst (hunk 2 above)
Comment (williamjamir) [reply docs/source/concepts/application_usage.rst]: Yep, removed ✔️

File: docs/source/quick_start.rst (hunk 3 above)
Comment (williamjamir) [reply docs/source/quick_start.rst]: 👍

**Pull request merged**